interface/patient_file/summary/pnotes_full.php

```diff
@@ -64,8 +64,8 @@
 $M = 15;
 
 $mode   = $_REQUEST['mode'];
-$offset = $_REQUEST['offset'];
-$offset_sent = $_REQUEST['offset_sent'];
+$offset = (int)$_REQUEST['offset'];
+$offset_sent = (int)$_REQUEST['offset_sent'];
```

Comment:
Much more secure and less bug prone to escape these elements downstream in the script.
At lines 512, 525, 667, 681
(note this is just being anal since they are being added to something and will be turned into an int anyways)

Reply:
And to clarify, would then not convert above to int. As I just posted on your colleagues PR, you just never know sometimes how a developer is going to use these elements. Sometime, they'll use a string to represent something (for example, 'all' in some cases to bypass using an offset).

Reply:
Done! according to your comments.
But in this case, a cast should be applied. Because some mathematical operations are done with this variable, so if you pass a text then you will have an error.
Unless I missed something :)

Reply:
In this case, an error is ok, since this is because somebody is doing something they should not be doing(and no real difference between an error or nonsensical output in this case). Goal is to prevent security vulnerabilities :)